### ISSUE TITLE
ns.formulas.hacking.growPercent is misleading

### ISSUE THREAD

**Issue by quyteriyaki** [danielyxie/bitburner#2678]
**Description:** Within documentation, it is stated that `ns.formulas.hacking.growPercent()` returns a percentage, where it actually returns a multiplier value in line with what is used for `ns.growthAnalyze`.
**Location of issue:** [Documentation for growPercent()](https://github.com/danielyxie/bitburner/blob/dev/markdown/bitburner.hackingformulas.growpercent.md)
(Relevant) [Documentation for growthAnalyze()](https://github.com/danielyxie/bitburner/blob/dev/markdown/bitburner.ns.growthanalyze.md)
**Steps to reproduce:**
```javascript
export async function main(ns) {
	const growthAmount = 2
	const threads = ns.growthAnalyze("n00dles", growthAmount)
	const percentage = ns.formulas.hacking.growPercent(ns.getServer("n00dles"), Math.round(threads), ns.getPlayer())
	ns.tprint(`Growth Amount: ${growthAmount}`)
	ns.tprint(`Number of threads : ${threads}`)
	ns.tprint(`Percentage of growth: ${percentage}`)
}
```
**Expected:** The terminal output log for "Percentage of growth" should result in the percentage of money increased within the server's balance relative to the server's previous balance. This would be inline with the output presented by `ns.grow()`. 
The value outputted is represented as a percentage in either decimal or percentage form. The scenario will output either `100%` or `1.00`.

**Actual:** The terminal output log for "Percentage of growth" outputs a value similar value to `growthAmount`. Further experimentation with `growthAmount` results in `percentage` returning similar values. From inference, the output of two functions have the same meaning. Further proof of correlation can be seen below. The equations used were sourced from relevant functions [numCycleForGrowth](https://github.com/danielyxie/bitburner/blob/8b69fd7faa5e29cd7d72f4bb8aad6e6c3f6cbf24/src/Server/ServerHelpers.ts#L49) and [calculateServerGrowth](https://github.com/danielyxie/bitburner/blob/8b69fd7faa5e29cd7d72f4bb8aad6e6c3f6cbf24/src/Server/formulas/grow.ts) respectively. Several variables were removed through simplification.

![image](https://user-images.githubusercontent.com/45784905/149763681-6d4a3d3a-5494-4297-94d0-4f9907b6890f.png)

**Suggested solution:**
1. Alter references of the term "percentage" to "multiplier" to put documentation inline with `ns.growthAnalyze()`.
2. In extension and entirely on the developer's discretion, implement a function of the same name `ns.formulas.hacking.growPercent()` which returns a value expected as indicated above.

**Comment by quyteriyaki** [comment]
Once again I clown on submitting issues without full examination on the issue itself.
A percentage can be acquired by subtracting 1. I would still like this open as the description of the function may infer different meanings.

**Comment by zeddrak** [comment]
Grow should grow by more than 100%. ie, 1.50 is 150% and means it would grow to 150% of current value. How should this be worded?

**Comment by zeddrak** [comment]
Addressed in PR #2771 (if I did that right.)

**Comment by Pongard** [comment]
Is this still a thing in Steam stable build? Confused the heck out of me, and led to incorrect calculations.
